actionpack/lib/action_dispatch/middleware/cookies.rb

```diff
@@ -389,6 +395,7 @@ def []=(name, options)
           options = { value: value }
         end
 
+        preprocess_options(options)
         handle_options(options)
```

Comment:
"We should probably just call preprocess_options in handle_options as well." Did you mean say: "delete the function call here and add it to handle_options?" :smile: